lib/apiaria.rb

```diff
@@ -28,4 +28,27 @@ def get_mentor_availability(mentor_id)
     JSON.parse(response.body)
   end
   
+  def get_messages(pageNum = nil)
+    unless pageNum
+      response = self.class.get("/message_threads",  headers: { "authorization" => @auth_token })
+      JSON.parse(response.body)
+    else
+      url = "/message_threads?page=" + pageNum.to_s
+      response = self.class.get(url,  headers: { "authorization" => @auth_token })
+      JSON.parse(response.body)
+    end
+  end
+
+  def create_message(sender, recipientId, body, options = {})
```

Comment:
@praveenyr there isn't a need to create an args hash here to pull keys out since the values get passed explicitly. Also, the token gets passed in via the @auth_token and we do need the text as an argument. 

An approach like this would be cleaner: 

```
def create_message(sender, recipientId, subject, text)
self.class.post(api_end_point("messages"),
      body: {
        "sender": sender,
        "recipient_id": recipientId,
        "subject": subject,
        "stripped-text": text
        },
      headers: {"authorization" => @auth_token})
end
```

p.create_message("praveenyr@gmail.com", 623967, "I need some Hawaii!", "Now!")